modules/tile-converter/test/i3s-converter/i3s-converter.spec.js

```diff
@@ -38,6 +38,32 @@ setLoaderOptions({
   _worker: 'test'
 });
 
+test('tile-converter - Converters#texure attributes', async (t) => {
```

Comment:
The idea was to remove this test because the testing the whole converter takes a lot of time and diminishes CI speed